planning/behavior_path_planner/src/behavior_path_planner_node.cpp

```diff
@@ -549,7 +549,8 @@ void BehaviorPathPlannerNode::run()
   if (!clipped_path.points.empty()) {
     path_publisher_->publish(clipped_path);
   } else {
-    RCLCPP_ERROR(get_logger(), "behavior path output is empty! Stop publish.");
+    RCLCPP_ERROR_THROTTLE(
```

Comment:
these logging change should be for another small change PR like
 feat: change verbose